jetstream/first-time-wallpaper-newtab.toml

```diff
@@ -15,3 +15,45 @@ select_expression = 'SUM(0)'
 select_expression = 'SUM(0)'
 [metrics.new_tab_enabled_on_new_tabs]
 select_expression = 'SUM(0)'
+
+
+[data_sources.main]
+from_expression = """(
+    SELECT
+        client_id,
+        DATE(submission_timestamp) AS submission_date
+    FROM `moz-fx-data-shared-prod.telemetry_stable.main_v5`
+    WHERE DATE(submission_timestamp) = '2025-03-04'
```

Comment:
Is the single date intended here so that it won't scan too much data?

Reply:
Yep, maybe it would be better to just do a filter that won't get any data at all, but this was easiest and I think will do the trick. Open to a better filter though!

Reply:
can we just do "select 0"?

Reply:
It needs the `client_id` and `submission_date` columns so that the `JOIN` conditions don't cause an error.